frontend/src/pages/dashboard.js

```diff
@@ -167,185 +168,201 @@ class Dashboard extends Component {
-                                <td>
-                                  <h6>
-                                    <CandidateStatus status={candidate.status} />
-                                  </h6>
-                                </td>
-                              )}
-
-                              {selects.includes('Year') && <td>{candidate.year}</td>}
-                              {selects.includes('Graduation Year') && (
-                                <td>{candidate.graduationDate}</td>
-                              )}
-                              {selects.includes('Roles') && (
-                                <td>
-                                  {candidate.role
-                                    .filter(role => this.state.filters.roles.includes(role))
-                                    .join(', ')}
-                                </td>
-                              )}
-                              {selects.includes('Major') && <td>{candidate.major}</td>}
-                              {selects.includes('Hours') && <td>{candidate.timeCommitment}</td>}
-
-                              {selects.includes('Links') && (
-                                <td>
-                                  <CandidateLinksBadge link={candidate.resumeID} text="Resume" />
-                                  <CandidateLinksBadge link={candidate.linkedIn} text="LinkedIn" />
-                                  <CandidateLinksBadge link={candidate.github} text="Github" />
-                                  <CandidateLinksBadge link={candidate.website} text="Website" />
-                                </td>
-                              )}
+        {this.state.candidates.length !== 0 && (
+          <div className="page-content-wrapper dashboard">
+            <Container fluid>
+              <Row>
+                <Col lg="2" md="3" className="ml-2">
+                  <FilterComponent />
+                </Col>
+                <Col lg="9" md="8">
+                  <Container>
+                    <Row>
+                      <InfoAlert />
+                    </Row>
+                    <Row>
+                      <Col sm={7}>
+                        <FormGroup>
+                          <Label htmlFor="search" />
+                          <Input
+                            type="search"
+                            id="search"
+                            value={this.state.search}
+                            placeholder="Search Candidates by Name or ID"
+                            onChange={this.handleSearchInput}
+                          />
+                        </FormGroup>
+                      </Col>
+                    </Row>
+                    <Row>
+                      <Table size="m" hover className="candidate-table">
+                        <thead>
+                          <tr>
+                            <th>#</th>
+                            {selects.includes('Name') && <th>Name</th>}
```

Comment:
Future: I know this was not within scope of your PR, so it's not a blocking change. However, can we condense all of these to a map on an array containing all the attributes (i.e. `['Name','Status','Year'].map((elem, idx) => conditionalRenderElement(elem))`
It would be great if we could also bring this out into a separate component.

Reply:
yes, we should refractor it to that!